packages/bson-transpilers/test/yaml/driver-syntax.yaml

```diff
@@ -72,6 +72,13 @@ tests:
                       'x': new NumberLong(0), 
                       'y': new ObjectId()
                     });"
+                php: |-
+                    // Requires the MongoDB PHP Driver
+                    // https://www.mongodb.com/docs/drivers/php/
+
+                    $client = new Client('mongodb://localhost:27017');
+                    $collection = $client->selectCollection('db', 'collection');
+                    $cursor = $collection->aggregate(['x' => 0, 'y' => new ObjectId()]);
```

Comment:
@gribnoysup: I realize this is outside the scope of this PR, but more generally I think it'd make sense to open an internal Compass ticket to revise these test expectations to _not_ create an invalid `aggregate()` call. At the very least, this can be changed to nest the current parameter within a `$match` pipeline stage.